src/topology/category/Profinite.lean

```diff
@@ -68,6 +68,71 @@ end Profinite
 @[simps, derive [full, faithful]]
 def Profinite_to_Top : Profinite ⥤ Top := induced_functor _
 
+namespace Profinite
+
+universe u
+
+/-- An explicit limit cone for a functor to `Profinite`. -/
+def limit_cone {J : Type u} [small_category J] (F : J ⥤ Profinite.{u}) : limits.cone F :=
+{ X :=
+  { to_Top := (Top.limit_cone' $ F ⋙ Profinite_to_Top).X,
+    is_compact := begin
+      erw ← compact_iff_compact_space,
+      dsimp [functor.sections],
+      apply is_closed.compact,
+      have : {u : Π (j : J), (F.obj j) | ∀ {j j' : J} (f : j ⟶ j'), (F.map f) (u j) = u j'} =
+        ⋂ (a b : J) (f : a ⟶ b), { u : Π (j : J), F.obj j | F.map f (u a) = u b }, by tidy,
+      rw this, clear this,
+      apply is_closed_Inter,
+      intros a,
+      apply is_closed_Inter,
+      intros b,
+      apply is_closed_Inter,
+      intros f,
+      let π : (Π (j : J), F.obj j) → F.obj a × F.obj b := λ x, (x a, x b),
+      change is_closed (π ⁻¹' { x | F.map f x.1 = x.2 }),
+      refine is_closed.preimage (continuous.prod_mk (continuous_apply _) (continuous_apply _)) _,
+      refine is_closed_eq _ continuous_snd,
+      continuity,
```

Comment:
```suggestion
      exact is_closed.preimage (by continuity) (is_closed_eq (by continuity) continuous_snd),
```

Reply:
It's interesting that `by continuity` can't solve `continuous_snd` here, even though `continuous_snd` has the attribute. I don't see what's going on. `continuity!` does work, however, so another possibility here is
```
refine is_closed.preimage _ (is_closed_eq _ _); continuity!